docs/guide/runtime-sessions-cookies.md

```diff
@@ -369,3 +364,34 @@ return [
 
 > Info: [[yii\web\Request::cookieValidationKey|cookieValidationKey]] is critical to your application's security.
   It should only be known to people you trust. Do not store it in the version control system.
+  
+## Security settings
+
+Both [[yii\web\Cookie]] and [[yii\web\Session]] support the following security flags:
+
+### httpOnly
+
+For better security, the default value of [[yii\web\Cookie::httpOnly]] and the 'httponly' parameter of 
+[[yii\web\Session::cookieParams]] is set to `true`. This helps mitigate the risk of a client-side script accessing 
+the protected cookie (if the browser supports it).
+You may read the [HttpOnly wiki article](https://www.owasp.org/index.php/HttpOnly) for more details.
+
+### secure
+The purpose of the secure flag is to prevent cookies from being send in clear text. If the browsers supports the
+secure flag it will only include the cookie when the request is send over a secure (TLS) connection.
+You may read the [SecureFlag wiki article](https://www.owasp.org/index.php/SecureFlag) for more details.
+
+### sameSite
+Starting at Yii 2.0.21 the [[yii\web\Cookie::sameSite]] setting is supported, this also requires PHP version 7.3.0 or higher.
+The purpose of the sameSite setting is to prevent CSRF (Cross-Site Request Forgery) attacks.
+If the browsers supports the sameSite setting it will only include the cookie according to the specified policy ('Lax' or 'Strict').
```

Comment:
```suggestion
If the browsers supports the sameSite setting it will only include the cookie according to the specified policy ('Lax' or 'Strict').
```
```suggestion
If the browser supports the `sameSite` setting it will only include the cookie according to the specified policy ('Lax' or 'Strict').
```